### DISCUSSION TITLE
Can we visualize an arrow?

### DISCUSSION THREAD

**Discussion by keiohta** [haosulab/SAPIEN#82] [Q&A] [answered]
Hi author, thank you very much for your continuous help.
I would like to visualize an arrow as the attached picture (Fig.1 in [UMPNet paper [Xu, 2021]](https://arxiv.org/pdf/2109.05668.pdf)) to show force direction, but am not sure what's the best way to do that. I think one way is to load a URDF of the arrow and change its position and orientation when taking a picture, but I haven't found a good arrow model (found [one](https://github.com/berkayalpcakal/tum-adlr-ss20-02/blob/master/multi_goal/envs/assets/arrow.urdf) but not so good). So, could you share your thoughts on how to visualize this kind of arrow?

Thank you in advance!

![image](https://user-images.githubusercontent.com/7234712/174825569-c9f49797-ec57-4ce5-8598-30fe7922a3ef.png)

**Comment by fbxiang** [comment, marked as answer]
Since arrows are not articulated, you can just add a static/kinematic actor with a visual shape arrow attached (see documentation on adding an actor). You can generate arrows with code or draw one in modeling software.

**Reply by keiohta** [reply]
@fbxiang thanks for your comment. I followed your advice and successfully implemented the arrow using the static actor. Thanks!
